modules/account-lib/src/coin/trx/utils.ts

```diff
@@ -148,6 +156,10 @@ export function decodeTransaction(hexString: string): RawData {
       contractType = ContractType.AccountPermissionUpdate;
       contract = exports.decodeAccountPermissionUpdateContract(rawTransaction.contracts[0].parameter.value);
       break;
+    case 'type.googleapis.com/protocol.TriggerSmartContract':
+      contractType = ContractType.TriggerSmartContract;
+      contract = exports.decodeTriggerSmartContract(rawTransaction.contracts[0].parameter.value);
```

Comment:
hmmm if there's multiple contracts, we should throw potentially since we don't really support it. i'm not sure. this is a risk right now

Reply:
actually this is contract agnostic.
I mean that we can process any contract call tx, not only the ones that we use on wbtc